app/controllers/android/bottomnavigation.js

```diff
@@ -4,75 +4,62 @@ import LayoutParams from 'android.widget.FrameLayout.LayoutParams';
 import ViewGroupLayoutParams from 'android.view.ViewGroup.LayoutParams';
 import Gravity from 'android.view.Gravity';
 
-const activity = new Activity(Ti.Android.currentActivity);
+$.win.activity.onCreate = () => {
+	// Create a new instance by passing the current activity
+	const activity = new Activity($.win.activity);
+	const bottomNav = new BottomNavigationView(activity);
 
-(function (container) {
-  // Create a new instance by passing the current activity
-  const bottomNav = new BottomNavigationView(activity);
+	// Set the layout (in Ti-language: width = fill, height = size, bottom = 0)
+	bottomNav.setLayoutParams(new LayoutParams(ViewGroupLayoutParams.MATCH_PARENT, ViewGroupLayoutParams.WRAP_CONTENT, Gravity.BOTTOM));
 
-  // Set the layout (in Ti-language: width = fill, height = size, bottom = 0)
-  bottomNav.setLayoutParams(new LayoutParams(ViewGroupLayoutParams.MATCH_PARENT, ViewGroupLayoutParams.WRAP_CONTENT, Gravity.BOTTOM));
-  
-  // Optional: Handle reselection
-  bottomNav.setOnNavigationItemReselectedListener(new BottomNavigationView.OnNavigationItemReselectedListener({
-    onNavigationItemReselected: (menuItem) => {
-      const message = 'Menu reselected to "' + menuItem.getTitle() + '"';
+	// Optional: Handle reselection
+	bottomNav.setOnNavigationItemReselectedListener(new BottomNavigationView.OnNavigationItemReselectedListener({
+		onNavigationItemReselected: (menuItem) => {
+			const message = 'Menu reselected to "' + menuItem.getTitle() + '"';
+			Ti.API.info(message);
+			$.stateLabel.text = message;
+			return true;
+		}
+	}));
 
-      Ti.API.info(message);
-      $.stateLabel.text = message;
+	// Optional: Handle selection
+	bottomNav.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener({
+		onNavigationItemSelected: (menuItem) => {
+			const message = 'Menu item "' + menuItem.getTitle() + '" selected!';
+			Ti.API.info(message);
+			$.stateLabel.text = message;
+			return true;
+		}
+	}));
 
-      return true;
-    }
-  }));
+	// Crazy stuff here! So how does this work?
+	// Instead of using R.menu.tabs and R.colors.tabs_background_color, we use the programmatic approach to receive them
+	// In the future, this may also be possible by Ti.App.Android.R.*, but thats not available so far.
+	bottomNav.inflateMenu(Ti.App.Android.R.menu.tabs);
```

Comment:
Update the comment above the change here, since you now are using the `Ti.App.Android.R` namespace properties?

Reply:
Good catch. I've just updated it now.